tools/building.py

```diff
@@ -648,7 +648,7 @@ def move_to_safe_7bit_ascii_filename(filename):
 
     # Exit and print final hint to get clearer output
     msg = f'closure compiler failed (rc: {proc.returncode}): {shared.shlex_join(cmd)}'
-    if not pretty:
+    if not settings.MINIFY_WHITESPACE:
```

Comment:
This and a few of the below look flipped to me? `not pretty` should be replaced with `settings.MINIFY_WHITESPACE` (without a `not` before it).

Reply:
Done